sdks/python/apache_beam/io/avroio.py

```diff
@@ -693,29 +727,44 @@ def beam_row_to_avro_dict(
     return lambda row: convert(row[0])
 
 
+# convert a beam atomic value to an avro atomic value
+# since numeric values are converted to unsigned in
+# avro_atomic_value_to_beam_atomic_value we need to convert
+# back to a signed number
+def beam_atomic_value_to_avro_atomic_value(avro_type: str, value):
```

Comment:
Nit: consider using docstring for function documentation. (same for other functions)
```suggestion
def beam_atomic_value_to_avro_atomic_value(avro_type: str, value):
"""convert a beam atomic value to an avro atomic value

since numeric values are converted to unsigned in
avro_atomic_value_to_beam_atomic_value we need to convert
back to a signed number.
```